python/tests/serializers/test_loading_plan.py

```
@@ -0,0 +1,272 @@
+        Comment.objects.create(post=post, author=author, text=f"comment {i}")
+        posts.append(post)
+    return author, posts
+
+
+@pytest.fixture
+def count_queries(monkeypatch):
+    """Count SQL statements on every thread (the ORM executor uses its own connections)."""
+    statements: list[str] = []
+    original = db_utils.CursorWrapper._execute
+
+    def counting_execute(self, sql, params, *ignored_wrapper_args):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `counting_execute`

(ANN202)